internal/scorecard/alpha/scorecard.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/operator-framework/operator-sdk/version"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+)
+
+var (
+	Log = logrus.New()
+)
+
+type Options struct {
+	Config       Config
+	Selector     labels.Selector
+	List         bool
+	OutputFormat string
+	Client       kubernetes.Interface
+}
+
+// RunTests executes the scorecard tests as configured
+func RunTests(o Options) error {
+	tests := selectTests(o.Selector, o.Config.Tests)
+
+	for i := 0; i < len(tests); i++ {
+		if err := runTest(tests[i]); err != nil {
+			return fmt.Errorf("test %s failed %s", tests[i].Name, err.Error())
+		}
+	}
+
+	return nil
+}
+
+// LoadConfig will find and return the scorecard config, the config file
+// can be passed in via command line flag or from a bundle location or
+// bundle image
+func LoadConfig(configFlag string) (Config, error) {
+	c := Config{}
+
+	// TODO handle getting config from bundle (ondisk or image)
+	_, err := os.Stat(configFlag)
+	if os.IsNotExist(err) {
+		return c, err
+	}
```

Comment:
We can probably remove this since `ioutil.ReadFile` will return this error if the file doesn't exist.

Reply:
removed.